xml/System/NotSupportedException.xml

```diff
@@ -136,7 +136,7 @@
   
 -   For any arguments that require pre-allocated memory. If these exist, you should pass a reference to an existing variable.  
   
--   That the names of the exported functions are correct. This can be verified with [DumpBin.exe](https://msdn.microsoft.com/library/c1h23y6c.aspx).  
+-   That the names of the exported functions are correct. This can be verified with [DumpBin.exe](/cpp/build/reference/dumpbin-reference?view=msvc-160).  
```

Comment:
```suggestion
-   That the names of the exported functions are correct. This can be verified with [DumpBin.exe](/cpp/build/reference/dumpbin-reference).  
```